packages/user-extension/schema/user-menu-bar.json

```diff
@@ -0,0 +1,15 @@
+{
+  "title": "User Menu Bar",
+  "description": "User Menu Bar settings.",
+  "jupyter.lab.toolbars": {
+    "TopBar": [
+      {
+        "name": "user-menu",
+        "rank": 100
```

Comment:
Wondering if the rank should be `>900` here? If the user menu is supposed to be positioned in the top right corner.

Reply:
Will double check on Binder once it has finished building.

Reply:
Ah it's the toolbar that is at rank `900`. But the spacer is at rank `50`.

Probably we'll want to update that part of the documentation as well then: https://jupyterlab.readthedocs.io/en/latest/extension/extension_points.html?#top-area